_rules/inline-link-in-paragraph-has-distinguishable-style-not-based-on-color-alone-88407d.md

```diff
@@ -0,0 +1,319 @@
+---
+id: 88407d
+name: Inline link has distinguishable style or content not based on color alone
```

Comment:
How can a distinction based on content rely on color? The distinction based on color has to do with styles more than with content. So, I propose flipping around the title a little bit so that it reads clearer

```suggestion
name: Inline link has distinguishable style not based on color alone
or content that indicates it is a link
```